pbj-core/gradle/scripts/clear-gradle-cache.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Cleans build files in a way that './gradlew clean' can only dream of.
+
+# The location were this script can be found.
+SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 || exit ; pwd -P )"
+
+{
+  cd "$SCRIPT_PATH"/../..
+
+  pkill -9 -f gradle
+
+  rm -rvf ~/.gradle
```

Comment:
:) Something tells me this script belongs to your personal `~/bin/` :) But I don't mind having it here.

Reply:
I just brought it across from services because I hit a spot where gradle was unhappy and needed to get wiped, but I can just remove it.

Reply:
OK, that dropped the number of files in the diff by 40.